src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -302,4 +305,8 @@ public Schema getSchema() throws SchemaStoreException {
 	public void setFilter(MaxwellFilter filter) {
 		this.filter = filter;
 	}
+
+	public Long getReplicationLag() {
+		return this.replicationLag;
+	}
```

Comment:
yeah. when i went down that road i ran into issues with the classes hanging on init during tests. i suppose that makes this the tail wagging the dog a bit. i'm not thrilled with it.